ParkingLot/DataModels/ParkingBoyModels.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+
+namespace ParkingLot.DataModels
+{
+    public class ParkingBoy
+    {
+        public ParkingBoy(Lot[] parkingLots, int id = 0)
+        {
+            this.Id = id;
+            this.ParkingLots = parkingLots;
+        }
+
```

Comment:
It seems a parking boy should have parking lots when it is created, may need to be modified to meet the later requirement, adding parking boy without parking lot to a manager.